docs/ssm-deployment-guide.md

```diff
@@ -198,7 +198,7 @@ For deploy standalone SSM, SSM will only start SSM server without SSM agents. Di
 
    ## SSM Service with multiple Agents
 
-Distribute `${SMART_HOME}` directory to SSM Server nodes and each Smart Agent nodes. Smart Agent can coexist with Hadoop HDFS Datanode. For better performance, We recommend to deploy one agent on each Datanode. Of course, Smart Agents on servers other than Datanodes and different numbers of Smart Agents than Datanodes are also supported.
+To deploy SSM to Smart Server nodes and Smart Agent nodes (if configured), you should enter into ${SMART_HOME} directory and type ./bin/install.sh. You can use --config <config-dir> to specify where SSM's config directory is.${SMART_HOME}/conf is the default config directory if the config option is not used.Smart Agent can coexist with Hadoop HDFS Datanode. For better performance, We recommend to deploy one agent on each Datanode. Of course, Smart Agents on servers other than Datanodes and different numbers of Smart Agents than Datanodes are also supported.
```

Comment:
It's better to move the content you added to the place which is after this sentence: If better performance is desired, we recommend to deploy one agent on each Datanode.